### PR TITLE
Add basekit from conda workflow

### PR DESCRIPTION
**Pull request by leshikus**
Subj

### CODE REVIEW

**Comment by leshikus** [comment]
The workflow passes, https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/8939272321

File: third_party/intel/backend/driver.py [hunk 1]
```diff
@@ -8,12 +8,17 @@
...
+include_dir = [os.path.join(dirname, "include")]
+
+oneapi_root = os.getenv("ONEAPI_ROOT")
+if oneapi_root:
+    include_dir += [
+        os.path.join(oneapi_root, "compiler/latest/include"),
+        os.path.join(oneapi_root, "compiler/latest/include/sycl")
+    ]
+
+conda_prefix = os.getenv("CONDA_PREFIX")
```
Comment (pbchekin) [inline third_party/intel/backend/driver.py:20]: Is it used?

File: .github/workflows/conda-basekit-build-test.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "5 2 * * *"
+
+permissions: read-all
+
+env:
+  BACKEND: XPU
```
Comment (pbchekin) [inline .github/workflows/conda-basekit-build-test.yml:11]: Not used, not required.

File: third_party/intel/backend/driver.py (hunk 1 above)
Comment (pbchekin) [inline third_party/intel/backend/driver.py:20]: Is it used?

File: .github/workflows/conda-basekit-build-test.yml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  TRITON_DISABLE_LINE_INFO: 1
+  GH_TOKEN: ${{ github.token }}
```
Comment (pbchekin) [inline .github/workflows/conda-basekit-build-test.yml:13]: Is it required on the top level? It can be set for a specific step only.

**Review by pbchekin — COMMENTED** [review]
Looks good, several minor questions to address.

File: third_party/intel/backend/driver.py (hunk 1 above)
Comment (leshikus) [reply third_party/intel/backend/driver.py:20]: no, it is not; will delete this

File: .github/workflows/conda-basekit-build-test.yml (hunk 3 above)
Comment (leshikus) [reply .github/workflows/conda-basekit-build-test.yml:13]: Fixed

File: third_party/intel/backend/driver.py (hunk 1 above)
Comment (leshikus) [reply third_party/intel/backend/driver.py]: Fixed (duplicate)

**Review by pbchekin — COMMENTED** [review]
Looks good!

**Review by pbchekin — APPROVED** [review]

**Pull request merged**